### PR TITLE
CCM-5756: create template page snags

### PR DESCRIPTION
**Pull request by alibombaye**
## Description

Changes the name of the button and the labels to be bold.

Screenshot 2024-11-13 at 14 11 11
Screenshot 2024-11-13 at 14 12 02
Screenshot 2024-11-13 at 14 12 13

## Type of changes

- [x] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [x] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: tests/test-team/template-mgmt-component-tests/sms/template-mgmt-create-sms-page.component.ts [hunk 1]
```diff
@@ -228,9 +228,7 @@ test.describe('Create SMS message template Page', () => {
 
       await createSmsTemplatePage.messageTextArea.fill(templateMessage);
 
-      await createSmsTemplatePage.clickContinueButton();
-
-      await page.getByRole('button', { name: 'Continue' }).click();
```
Comment (alibombaye) [inline tests/test-team/template-mgmt-component-tests/sms/template-mgmt-create-sms-page.component.ts:233]: ???

File: tests/test-team/template-mgmt-component-tests/email/template-mgmt-create-email-page.component.ts [hunk 2]
```diff
@@ -221,9 +221,7 @@ test.describe('Create Email message template Page', () => {
 
       await createEmailTemplatePage.messageTextArea.fill(templateMessage);
 
-      await createEmailTemplatePage.clickContinueButton();
-
-      await page.getByRole('button', { name: 'Continue' }).click();
```
Comment (alibombaye) [inline tests/test-team/template-mgmt-component-tests/email/template-mgmt-create-email-page.component.ts:226]: ???

File: tests/test-team/template-mgmt-component-tests/sms/template-mgmt-create-sms-page.component.ts (hunk 1 above)
Comment (bhansell1) [reply tests/test-team/template-mgmt-component-tests/sms/template-mgmt-create-sms-page.component.ts:233]: There is actually a bug in this test.

If you update the playwright timeout and run the app in a debug terminal you'll see it. 

when browser clicks the continue button the web-app takes a few seconds to handle redirecting to a different page.

When the assertions for inputs are executed they happen on the same page before the app has had time to redirect the user and the test passes, which is wrong. 

What the test _should_ be doing (this is the same for Email). Is;

1. Loading a session with no data
2. Entering data
3. Clicking the continue button (save the data is saved)
4. Return back to CreateEmailPage 
5. Assert the inputted data is correct

Comment (alibombaye) [reply tests/test-team/template-mgmt-component-tests/sms/template-mgmt-create-sms-page.component.ts:233]: Updated to click on the back link after clicking on the Continue button

File: tests/test-team/template-mgmt-component-tests/email/template-mgmt-create-email-page.component.ts (hunk 2 above)
Comment (alibombaye) [reply tests/test-team/template-mgmt-component-tests/email/template-mgmt-create-email-page.component.ts:226]: Updated to click on the back link after clicking on the Continue button

**Review by chris-elliott-nhsd — APPROVED** [review]

**Review by bhansell1 — DISMISSED** [review]

**Review by jazrattanpal — DISMISSED** [review]

**Review by bhansell1 — APPROVED** [review]

**Review by jazrattanpal — APPROVED** [review]

**Pull request merged**